### ISSUE TITLE
Choose sides for buttons with isetekbd

### ISSUE THREAD

**Issue by anjiro** [terrychou/iVim#195]
I'm using ivim on an iPad with the floating on-screen keyboard, mostly with one thumb. It would be really nice to be able to add a ctrl/esc button on the right, but it seems that I can't specify the side the button should be on. The workaround is to make an extra "dummy" button for the other side, but somehow specifying side explicitly would be nice.

**Comment by terrychou** [comment]
Have you ever tried the `replace` operation `:help isetekbd-op-replace`?

**Comment by anjiro** [comment]
Yes, but I want a layout that's not balanced in left/right numbers. For example, exactly the same as the default set of buttons, but with an additional **esc/ctrl** and **tab/arrows** button on the right.

**Comment by terrychou** [comment]
I think with the available operations such as `remove` or `insert`, you can get to "move" the `esc/ctrl` and `tab/arrows` buttons to any locations as you like. Just keep in mind that the buttons are located by indexes other than sides.
